TriblerGUI/widgets/trustpage/js/main.js

```diff
@@ -47,6 +59,8 @@ function update(graph) {
 
     console.log(graph);
```

Comment:
Do we still need this debugging functionality here? If yes please add a string with explanation indicating what is logged.